packages/asset-uploader/src/helpers/image-fs-path.ts

```diff
@@ -0,0 +1,18 @@
+import path from "path";
+import appRoot from "app-root-path";
+import { fsEnvVariables } from "../schema";
+const fsUploadVars = fsEnvVariables.parse(process.env);
+
+export const imageFSDirectory = () => {
+  const basePath =
+    process.env.NODE_ENV === "development"
+      ? path.join(
```

Comment:
please avoid ternaries when you can, this is a perfect example where you could have returned early instead

Reply:
I am not returning early, I am deploying this to add the other part of the if

Reply:
but you can

Reply:
i was still testing, it's changed now

I needed to add the version for prod